src/main/java/org/team1540/robot2018/commands/JoystickDrive.java

```diff
@@ -0,0 +1,23 @@
+package org.team1540.robot2018.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.team1540.robot2018.OI;
+import org.team1540.robot2018.Robot;
+
+public class JoystickDrive extends Command {
+  public JoystickDrive() {
+    requires(Robot.drivetrain);
+  }
+
+  @Override
+  protected void execute() {
+    double triggerValue = OI.getDriverRightTrigger() - OI.getDriverLeftTrigger();
+    Robot.drivetrain.setLeft(-OI.getDriverLeftY() + triggerValue);
```

Comment:
Should the negation be done here on in `getDriver_()`?

Reply:
Here. Randomly negating things in `OI` is just confusing.